AUTHORS

```diff
@@ -0,0 +1,19 @@
+Authors sorted by their family-name.
+
+# Directly affiliated with EPFL/DEDIS
+
+Patricia Egger <patricia.egger@epfl.ch>
+Bryan Ford <bryan.ford@epfl.ch>
+David Froelicher <david.froelicher@epfl.ch>
+Nikitin Kirill <kirill.nikitin@epfl.ch>
+Kokoris Kogias Eleftherios <eleftherios.kokoriskogias@epfl.ch>
+Ismail Khoffi <ismail.khoffi@epfl.ch>
+Nicolas Gailly <nicolas.gailly@epfl.ch>
+Linus Gasser <linus.gasser@epfl.ch>
+Philipp Jovanovic <philipp.Jovanovic@epfl.ch>
+
+# NON-EPFL/DEDIS - signed a CLA / CLAI:
+
+Christian Mouchet <christian.mouchet@epfl.ch>
+Evan Visher <evan.visher@gmail.com>
```

Comment:
His address is already public. I only copied it from the git-log to the `AUTHORS`-file. So I don't see the issue.